Makefile

```diff
@@ -16,22 +16,41 @@ TARGET = cc-runtime
 DESTDIR :=
 PREFIX := /usr/local
 BINDIR := $(PREFIX)/bin
-QEMUBINDIR := $(BINDIR)
 SYSCONFDIR := $(PREFIX)/etc
 LIBEXECDIR := $(PREFIX)/libexec
 LOCALSTATEDIR := $(PREFIX)/var
 SHAREDIR := $(PREFIX)/share
 
 CCDIR := clear-containers
+
 PKGDATADIR := $(SHAREDIR)/$(CCDIR)
+PKGLIBDIR := $(LOCALSTATEDIR)/lib/$(CCDIR)
+PKGRUNDIR := $(LOCALSTATEDIR)/run/$(CCDIR)
 PKGLIBEXECDIR := $(LIBEXECDIR)/$(CCDIR)
 
+KERNELPATH := $(PKGDATADIR)/vmlinux.container
+IMAGEPATH := $(PKGDATADIR)/clear-containers.img
+
+QEMUBINDIR := $(BINDIR)
+QEMUCMD := qemu-lite-system-x86_64
+QEMUPATH := $(QEMUBINDIR)/$(QEMUCMD)
+
+SHIMCMD := cc-shim
```

Comment:
This looks a bit too much, eg. I don't see a need to provide compile time options for SHIMPATH beyond PKGLIBEXECDIR. The default value in the binary can be `"pkgLibexeDir" + "/cc-shim"`, still configurable at run-time through the config file.